docs/source/conf.py

```diff
@@ -66,7 +66,6 @@
 from pygments.lexers.web import PhpLexer
 lexers["php"] = PhpLexer(startinline=True, linenos=1)
 lexers["php-annotations"] = PhpLexer(startinline=True, linenos=1)
-primary_domain = "php"
```

Comment:
This was throwing an error in my build, and some googling suggested it was not necessary. Certainly doesn't seem to change anything?